include/ddc/kernels/splines/spline_builder.hpp

```diff
@@ -82,11 +82,12 @@ class SplineBuilder
     std::unique_ptr<ddc::detail::Matrix> matrix;
 
 private:
+    double m_dx; // average cell size for normalization of derivatives
+
     const int m_offset;
 
     interpolation_domain_type m_interpolation_domain;
 
-    double m_dx; // average cell size for normalization of derivatives
```

Comment:
Can you revert the changes ? This modifies the order of initialization of the member variables